students/julienjass/Lesson07/html_render.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python3
+
+class Element:
+    """Base Class of the HTML Render"""
+    tag = 'html'
+    ind = 0
+
+    def __init__(self, content=None, **kwargs):
+        if content is None:
+            self.content = []
+        if content != None:
+            self.content = [content]
+        if kwargs:
+            self.attrs = kwargs
+        else:
+            self.attrs = None
+
+    def append(self, new_content):
+        """Add content to Element"""
+        self.content.append(new_content)
+
+    def add_newlines(self):
+        """Loop thru self.content and insert newlines"""
+        if any(isinstance(line, str) for line in self.content):
+            content_copy = self.content[:]
+            for i in range(len(content_copy)-1):
+                next_index = i + 1
+                if isinstance(content_copy[i], object) and not isinstance(content_copy[i], str):
+                    pass
+                elif not isinstance(content_copy[next_index], str):
+                    self.content.insert(next_index, '\n')
+        if isinstance(self.content[-1], str):
+            self.content.append('\n')
+
+    def render(self, out_file, **kwargs):
+        """Render content and attributes with correct level of indentation"""
+        self.add_newlines()
+        if 'indent' in kwargs:
+            indent = kwargs['indent']
+        else:
+            indent = self.ind
+        if self.attrs:
+            attr_string = ''
+            for key, value in self.attrs.items():
+                attr_string += str('{}="{}" '.format(key, value))
+            out_file.write(f'{" "*indent}<{self.tag} {attr_string.strip()}>\n')
+        else:
+            out_file.write(f'{" "*indent}<{self.tag}>\n')
+        for line in self.content:
+            if isinstance(line, str):
+                out_file.write(f'{" "*(indent+4)}{line}')
+            elif isinstance(line, object):
```

Comment:
Everything is an object in Python so this isn't really doing anything, I would just go with an `else` clause to catch objects that aren't strings